docs/http/README.md

```diff
@@ -2,25 +2,25 @@
 
 ## Architecture
 
-Dans une *Single Page Application (SPA)*, la communication avec le serveur se fait via des requêtes HTTP asynchrones (*AJAX*) ou des protocoles plus spécialisés comme les WebSocket. Nous allons voir comment faire ces requêtes réseau depuis une application Vue.
+In a *Single Page Application (SPA)*, communication with the server is via asynchronous HTTP requests (*AJAX*) or more specialized protocols such as WebSocket. We will see how to make these network requests from a Vue application.
 
-Vue.js est un framework qui se focalise sur l'interface utilisateur et ne propose rien de particulier en ce qui concerne les échanges avec le serveur, préférant laisser ce choix d'implémentation au développeur. Le moyen le plus simple de lancer une requête HTTP asynchrone en JavaScript est d'utiliser [la méthode `fetch`](https://developer.mozilla.org/fr/docs/Web/API/Fetch_API/Using_Fetch). Elle n'est pas supportée sur Internet Explorer mais il existe des polyfills pour compléter le support. Comme alternative, vous pouvez passer par d'autres bibliothèques tierces plus complètes telles que [Axios](https://github.com/axios/axios), qui est recommandée par l'équipe de Vue.
+Vue.js is a framework that focuses on the user interface and offers nothing special to exchanges with a server. this choice of implementation is left to the developer. The easiest way to run an asynchronous HTTP request in JavaScript is to use [the `fetch` method](https://developer.mozilla.org/en/docs/Web/API/Fetch_API/Using_Fetch). It is not supported on Internet Explorer but there are polyfills to complete the support. As an alternative, you can use other more complete third-party libraries such as [Axios](https://github.com/axios/axios), which is recommended by the Vue team.
 
 ::: tip
-On évitera de faire les appels réseau directement depuis le code d'un composant. Cela empêche de réutiliser le code facilement si un autre composant doit faire le même appel. Optez plutôt pour systématiquement séparer la logique applicative de la logique de vos vues.
+Network calls should be avoided directly from the code of a component. This prevents reuse of the code easily if another component has to make the same call. Instead, opt to systematically separate the application logic from the logic of your views.
```

Comment:
should be avoided directly => should not be done directly

(avoid directly sonne bizarre)